pennylane/io.py

```diff
@@ -99,12 +106,14 @@ def from_qiskit(quantum_circuit):
 
     Args:
         quantum_circuit (qiskit.QuantumCircuit): a quantum circuit created in qiskit
+        measurements (list[MeasurementProcess]): the list of PennyLane measurements that
+            overrides the terminal measurements that may be present in the imput circuit.
 
     Returns:
         function: the PennyLane template created based on the QuantumCircuit
```

Comment:
Can you add a **Details** section below that shows an example using the `measurements` argument?

Reply:
Added!